include/fmt/std.h

```diff
@@ -448,5 +448,18 @@ struct formatter<std::atomic<T>, Char,
   }
 };
 
+#if defined(__cpp_lib_atomic_flag_test) && __cpp_lib_atomic_flag_test >= 201907L
```

Comment:
I observe the other feature checks don't check for the feature value, shouldn't check here too?

Reply:
No, I mean make like `#ifdef __cpp_lib_optional` above without comparing the value, as we're looking for the first and (currently) the only version of the feature

Reply:
I think it's enough to check if `__cpp_lib_atomic_flag_test` is defined unless this needs to rely on some specific change to `atomic_flag` which is unlikely.